CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ Usage:
 ./scripts/extract-changelog-for-version.sh 1.3.37 5
 ```
 ### 0.2.1-SNAPSHOT
+* Port [#1794](https://github.com/fabric8io/fabric8-maven-plugin/pull/1794): Fix ImageChange triggers not being set in DeploymentConfig when resource fragments are used.
+
```

Comment:
This will break the changelog parser and won't look good in an e-mail
```suggestion
* Fix #97:  Port of fabric8io/fabric8-maven-plugin#1794 to fix ImageChange triggers not being set in DeploymentConfig when resource fragments are used

```